README.md

```diff
@@ -1,81 +1,46 @@
-Airport Challenge
-=================
+in the file index.js it is determined if there is stormy weather or not, using the Math.random function. In the tests, the weather variable is set as required for the tests.
 
-```
-         ______
-        __\____\___
-=  = ==(____DFA____)
-           \_____\__________________,-~~~~~~~`-.._
-          /     o o o o o o o o o o o o o o o o  |\_
-          `~-.__       __..----..__                  )
-                `---~~\___________/------------`````
-                =  ===(_________)
-
-```
-
-Instructions
----------
-
-* Feel free to use google, your notes, books, etc. but work on your own.
-* Keep it SIMPLE - it's not nearly as complicated as it first may look.
-* You must [submit your challenge](https://airtable.com/shrUGm2T8TYCFAmjN) by the deadline, wherever you get to.
-* Use your own test framework and evidence your test-driven development by committing on passing tests.
-* Please write your own README detailing how to install your project, how to run the tests, how you approached the problem and provide screenshots of interacting with your program.
-* If you refer to the solution of another coach or student, please put a link to that in your README.
-* Please create separate files for every class, module, and spec.
-
-Steps
--------
-
-1. Fork this repo, and clone to your local machine
-2. `npm install` to install project dependencies
-3. Convert stories into a representative domain model and test-drive your work.
-4. Run your tests using `npm test` or `node specRunner.js`
-5. OPTIONAL: [Lint](https://eslint.org/docs/user-guide/getting-started) your source code using `npx eslint src`.
-
-Task
------
-
-We have a request from a client to write the software to control the flow of planes at an airport. The planes can land and take off provided that the weather is sunny. Occasionally it may be stormy, in which case no planes can land or take off.  Here are the user stories that we worked out in collaboration with the client:
-
-#### Acceptance Criteria
-```
+story 1 (test 1)
 As an air traffic controller
 So I can get passengers to a destination
 I want to instruct the airport to land a plane
 
+story 2 (test 2)
 As the system designer
 So that the software can be used for many different airports
 I would like a default airport capacity that can be overridden as appropriate
 
+story 3 (test 3)
 As an air traffic controller
 To ensure safety
 I want to prevent landing when the airport is full
 
+story 4 (test 4)
 As an air traffic controller
 So I can get passengers on the way to their destination
 I want to instruct the airport to let a plane take off and confirm that it is no longer in the airport
 
+Story 5 (test 5)
+As an air traffic controller
+To avoid confusion
+I want the airport to only give a takeoff clearance to airplanes that are at the airport
+
```

Comment:
glad I split this user story, made it much easier to deal with, as I am still in a practical state of mind regarding airports, was sometimes difficult to strip that away and jut look at the logic